SU-SOE/soe_profile
# Jack 302 flatten profile

# NOT READY FOR REVIEW

# Summary
- Initial Flattening of SOE profile

# Review Needed By (Date)
- 1-23-2020

# Urgency
- Critical

# Steps to Test

1. After grabbing this branch of lelandd8 https://github.com/SU-SWS/acsf-lelandd8/tree/JACK-302 and setting up blt, etc. do a composer install --prefer-source
2. Checkout this branch on the soe profile: JACK-302-flatten-profile
3. checkout this branch of stanford_news: https://github.com/SU-SOE/stanford_news/tree/mods-additions
4. run blt drupal:install
5. verify it successfully installed without errors

# Affected Projects or Products
- Leleandd8

# Associated Issues and/or People
- https://github.com/SU-SWS/acsf-lelandd8/tree/JACK-302
- https://github.com/SU-SOE/stanford_news/tree/mods-additions
